nogadas_vonryan/index.html

```diff
@@ -108,34 +108,31 @@ <h2>My Goals</h2>
 			</ul>
 			<div class="comment-section">
 				<h2>Comment Section</h2>
-				<ul>
-					<li>Your goals are very ambitious! -Jhef</li>
-					<li>I hope you achieve all your goals that will 
-						help you be successful in life - Andrea</li>
-					<li>Keep on going hard work pays off - Mau</li>
-					<li>Your perspective is refreshing - Mark</li>
-				</ul>
+				<select id="sort_type" onchange="sortComments()">
+					<option value="newest_first">Newest First</option>
```

Comment:
by checking your Netlify link, I notice that the "Newest First" option displays the oldest comments, while the "Oldest First" displays the new ones.

Reply:
```fix inverted sorting function ``` - cda39b93b6c9b7f57778070850e666339ae5dbca

Netlify link was updated. Thank you po!